kubernetes/README_en.md

```diff
@@ -458,6 +458,8 @@ cp api-platform-hybrid/kubernetes/helm/values_examples/api-authorization/values.
 
 Alter the parameter values of the file ``api-platform-hybrid/api-gateway.yaml`` according to the instructions of the section **Changing Modules Versions and Other Parameters**.
 
+> NOTE: The property ``apigateway_label`` must be filled in with the desired name for the Gateway Pool, and informed later through the opening of a ticket to ** Support and Operations ** team can complete the creation.
```

Comment:
```suggestion
> NOTE: The property ``apigateway_label`` must be filled in with the desired name for the Gateway Pool and informed later through the opening of a ticket so that the **Support and Operations** team can conclude the creation.
```